toplevel/native/topeval.ml

```diff
@@ -32,6 +32,8 @@ let _dummy = (Ok (Obj.magic 0), Err "")
 external ndl_run_toplevel: string -> string -> res
   = "caml_natdynlink_run_toplevel"
 
+let implementation_label = " - native toplevel"
```

Comment:
I'm sorry to be that nitpicking guy again, but I would use `Some "native toplevel"` and decide on the dash at render-time.